scss/bitstyles/components/dropdown-menu.stories.mdx

```diff
@@ -0,0 +1,161 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Dropdown menu" />
+
+# Navbar
+
+A top-level navigation container, with two sections separated to the left & right. the left-hand side is commonly used for a logo and links, the right-hand side for user account menu, and global search.
```

Comment:
In addition to that, I don't fully get the "left and right" thing 🙈 

<img width="1152" alt="Screenshot 2021-02-18 at 10 22 05" src="https://user-images.githubusercontent.com/3491815/108335368-294ecb80-71d3-11eb-9228-d32c833f95da.png">

Where is left and where is right? The two items are underneath each other 🤔

Reply:
Same here, copy/paste error, will investigate 👍